src/utils/formatting.js

```diff
@@ -10,16 +10,19 @@ import { CHAT_SEEN_KEY, CID } from '../config/env';
  * @property {string} message - The content with options removed.
  * @property {Array<{id: string, label: string, value: string}>} options - An array of extracted options as objects.
  */
-export const extractOptionSet = (val) => {
+export const getExtractedData = (val, filterOptions = ['email_intent', 'payment_intent', 'MerchantSite', 'NewSupportTicket']) => {
   const match = /\[(.*?)\]/.exec(val);
 
   if (!match || match.length <= 1) {
     return { content: val, options: [] };
   }
 
+  const extractedOptions = match[1].split('|');
+
   return {
     content: val.replace(match[0], ''),
-    options: match[1].split('|').map((item, index) => ({ id: `opt-${index}`, label: item, value: item })),
+    options: extractedOptions.map((item, index) => ({ id: `opt-${index}`, label: item, value: item })).filter(item => !filterOptions.includes(item.value)),
+    filtered: extractedOptions.filter((item) => filterOptions.includes(item)),
   };
 };
 
```

Comment:
@IlievaDayana You can add some tests for `getExtractedData()` if there aren't any.

Reply:
they aren't any as unit, only as integration